src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
+import { repeatArray } from '../../services/utils'
+import { User } from './../../pages/users/types'
+
+export const users = repeatArray<User>(50, [
+  {
```

Comment:
Sorting should respect all items regardless of pagination

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/bdfc4e8e-51fb-47ae-bf51-c79f1facbe1b)